lib/ansible/modules/cloud/amazon/rds_param_group.py

```diff
@@ -109,64 +115,56 @@
       engine: 'mysql5.6'
       params:
           auto_increment_increment: "42K"
+      tags:
+          Environment: production
+          Application: parrot
 
 # Remove a parameter group
 - rds_param_group:
       state: absent
       name: norwegian_blue
 '''
 
-VALID_ENGINES = [
-    'aurora5.6',
-    'mariadb10.0',
-    'mariadb10.1',
-    'mysql5.1',
-    'mysql5.5',
-    'mysql5.6',
-    'mysql5.7',
-    'oracle-ee-11.2',
-    'oracle-ee-12.1',
-    'oracle-se-11.2',
-    'oracle-se-12.1',
-    'oracle-se1-11.2',
-    'oracle-se1-12.1',
-    'postgres9.3',
-    'postgres9.4',
-    'postgres9.5',
-    'postgres9.6',
-    'sqlserver-ee-10.5',
-    'sqlserver-ee-11.0',
-    'sqlserver-ex-10.5',
-    'sqlserver-ex-11.0',
-    'sqlserver-ex-12.0',
-    'sqlserver-se-10.5',
-    'sqlserver-se-11.0',
-    'sqlserver-se-12.0',
-    'sqlserver-web-10.5',
-    'sqlserver-web-11.0',
-    'sqlserver-web-12.0',
-]
-
-try:
-    import boto.rds
-    from boto.exception import BotoServerError
-    HAS_BOTO = True
-except ImportError:
-    HAS_BOTO = False
-
+RETURN = '''
+db_parameter_group_name:
+    description: Name of DB parameter group
+    type: string
+    returned: when state is present
+db_parameter_group_family:
+    description: DB parameter group family that this DB parameter group is compatible with.
+    type: string
+    returned: when state is present
+db_parameter_group_arn:
+    description: ARN of the DB parameter group
+    type: string
+    returned: when state is present
+description:
+    description: description of the DB parameter group
+    type: string
+    returned: when state is present
+errors:
+    description: list of errors from attempting to modify parameters that are not modifiable
+    type: list
+    returned: when state is present
+tags:
+    description: dictionary of tags
+    type: dict
+    returned: when state is present
+'''
```

Comment:
might be worth considering returning parameters that aren't the default. More obvious what things have been changed.